api/handler.go

```diff
@@ -163,6 +166,43 @@ func streamResults(results chan *events.Event, w http.ResponseWriter, r *http.Re
 	}
 }
 
+func requestResults(w http.ResponseWriter, r *http.Request) error {
+	oldUrlString := r.URL.RequestURI()
+	glog.Infof("oldUrlString is %v", oldUrlString)
+	requestElements := clientRegexp.FindStringSubmatch(oldUrlString)
+	if len(requestElements) != 3 {
+		return fmt.Errorf("Could not interpret client request: %v", requestElements)
+	}
+	newUrlString := "http://localhost:8080" + requestElements[1] + "events" + requestElements[2]
+	glog.Infof("requestElements are %v", requestElements)
+	glog.Infof("newUrlString is %v", newUrlString)
+	req, err := http.NewRequest("GET", newUrlString, nil)
```

Comment:
Sorry, I don't have enough context as to what client API is supposed to do. From here, it looks like its rewriting the query to events endpoint and just print out the result. I haven't followed the events stuff in detail before. Can you describe it a bit here.